securedrop/client.py

```diff
@@ -37,31 +40,86 @@ def make_dict(self):
         }
 
 
+class AESWrapper(object):
+
+    def __init__(self, key):
+        self.bs = AES.block_size
+        self.key = hashlib.sha256(key.encode()).digest()
+
+    def encrypt(self, raw):
+        verify = hashlib.sha256((raw.encode())).hexdigest()
+        raw = self._pad(raw)
+        iv = Random.new().read(AES.block_size)
+        cipher = AES.new(self.key, AES.MODE_CBC, iv)
+        bites = base64.b64encode(iv + cipher.encrypt(raw.encode()))
```

Comment:
You spelled "bytes" slightly wrong :)

Reply:
It was yelling at me for using bytes as that's already a type!

Reply:
trailing_ is apparently the convention for caseslike this.